src/lib/server/s3.ts

```diff
@@ -1,4 +1,3 @@
-import { building } from '$app/environment';
```

Comment:
Was used to avoid errors when `pnpm build` on the project. Not sure if still relevant

See https://svelte.dev/docs/kit/building-your-app